config/conf.d/global.conf

```diff
@@ -0,0 +1,9 @@
+global: {
+    # Path to logback logging configuration file; for more
+    # info, see http://logback.qos.ch/manual/configuration.html
+    logging-config: /etc/puppetserver/logback.xml
+}
+
+product: {
+    name: puppetserver
+}
```

Comment:
So, you're creating a new directory called `config` at the root, to put the ezbake stuff into.  Questions about this...

1. Do these files overlap with the ones in resources/ext/ezbake?  I think this basically boils down to the difference between composite ezbake builds and single-project ezbake builds, right?  If so, can we put a bit more thought into directory naming conventions to try to make the distinction more obvious?
2. Can we maybe call the top-level dir 'ezbake' to make it more obvious?
3. Should that path be configurable in the ezbake plugin settings?  (but still default to something explicit, like 'ezbake'?)